src/routes/create.js

```diff
@@ -1,7 +1,38 @@
+const Rooms = require('../../rooms.js');
+const Room = require('../../room.js');
+
 module.exports = {
+  method: 'POST',
   path: '/create',
-  method: 'get',
-  handler : (req, res) => {
-    // create route
+  handler: (req, reply) => {
+    // Verify username and password - in a later sprint
+    // if (user is authenticated) {
+    let username = req.body.username;
+    let roomName = req.body.roomName;
+    let roomId = Math.random().toString(36).slice(2);
+    roomId = new Room(username, roomName, permissions);
+
+    let endpointId = Room.roomId.createEndpointId();
```

Comment:
instead of the createEndpointId method here, uses a similar function above to create a random endpointId;
```
 Math.random().toString(36).slice(2);
```